yb-voyager/src/query/queryparser/traversal_plpgsql.go

```diff
@@ -196,7 +196,10 @@ func formatExprQuery(q string) string {
 		 "PLpgSQL_expr": {
 			"query": "'DROP TABLE IF EXISTS employees'",
 	*/
-	q = strings.Trim(q, "'") //to remove any extra '' around the statement
+	//to remove any extra '' around the statement
```

Comment:
context? trim didn't work?

Reply:
can't use trim as we don't want any other single quote in the statement to be removed just the first and last one if both are there. e.g. `NOTIFY my_table_changes, 'New row added with name: Charlie'`